index.html

```diff
@@ -18,21 +18,46 @@
         <h1>Mehmet Akif Tütüncü</h1>
         <h2>Computer Engineer</h2>
       </div>
+      <div class="cv">
+        <a class="link" href="cv.pdf" target="_blank">
+          📄<span class="link-text">CV.pdf</span>
+        </a>
+        <a class="link" href="cv.json" target="_blank">
+          ⚙️<span class="link-text">CV.json</span>
+        </a>
+      </div>
       <div class="bio">
         <span class="bio-row">
-          <span class="bio-emoji">👨🏻‍💻</span>Senior Software Engineer at <a class="inline-content" href="https://sahibinden.com" target="_blank"><img class="bio-icon" src="https://cdn.jsdelivr.net/npm/simple-icons@latest/icons/sahibinden.svg"/>sahibinden.com</a>
+          <span class="bio-emoji">👨🏻‍💻</span>
+          <span class="bio-title">Senior Software Engineer at  <a href="https://sahibinden.com" target="_blank"><img class="bio-icon center" src="https://cdn.jsdelivr.net/npm/simple-icons@latest/icons/sahibinden.svg"/>sahibinden.com</a></span>
         </span>
         <span class="bio-row">
-          <span class="bio-emoji">🎓</span>Izmir Institute of Technology
+          <span class="bio-emoji">🎓</span>
+          <span class="bio-title">Izmir Institute of Technology</span>
         </span>
         <span class="bio-row">
-          <span class="bio-emoji">❤️</span>Open source and functional programming advocate
+          <span class="bio-emoji">❤️</span>
+          <span class="bio-title">Open source and functional programming advocate</span>
         </span>
         <span class="bio-row">
-          <span class="bio-emoji">⌨️</span>Writes<span class="inline-content"><img class="bio-icon" src="https://cdn.jsdelivr.net/npm/simple-icons@latest/icons/scala.svg"/>Scala</span><span class="inline-content"><img class="bio-icon" src="https://cdn.jsdelivr.net/npm/simple-icons@latest/icons/kotlin.svg"/>Kotlin</span><span class="inline-content"><img class="bio-icon" src="https://cdn.jsdelivr.net/npm/simple-icons@latest/icons/java.svg"/>Java</span>
+          <span class="bio-emoji">⌨️</span>
+          <span class="bio-title">Writes</span>
         </span>
         <span class="bio-row">
-          <span class="bio-emoji">😻</span>Human of 1 (one) cat
+          <span class="bio-emoji"><img class="bio-icon" src="https://cdn.jsdelivr.net/npm/simple-icons@latest/icons/scala.svg"/></span>
```

Comment:
I don't think we should separate Scala, Kotlin, Java into different elements. They should be in one line (in one `bio-row`).

Reply:
Yes, since each of them is an item, it would be more correct for them to have a new line for consistency.

Reply:
> Another minor thing: some fonts are much smaller now, it will be much better if things stay mostly the same size. Thanks. 😊

I agree on the consistency. But I disagree on readability. Different size fonts are required for different sized screens.

Reply:
I don't mean being on the same line in the source code, I mean they should be on the same line on the rendered page. 😊